### DISCUSSION TITLE
“Actors” dropdown menu shouldn’t be an editable text area

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#142] [General]
**Bug description**
The user can input text into the area intended to function as a simple dropdown menu.

**Steps to reproduce**
1. Launch Tero Subtitler.
2. Click on `New empty subtitle` on startup.
3. Place the mouse cursor inside the “Actor” dropdown menu.
4. Write something inside the text area (unwanted behaviour).

**Expected behaviour**
The user should only be able to select the items from the dropdown menu after entering them in the dialogue box that opens by clicking on the icon with the silhouette of a profile.

**Screenshots/Video**
https://github.com/URUWorks/TeroSubtitler/assets/16348719/8b255aaa-ffad-4231-8629-b5173088e905

**Info**
 - MacBook Air M2 2022, running macOS Ventura 13.4.1 (c)
 - Tero Subtitler 1.0.0.7

**Comment by URUWorks** [comment]
It's not really a bug, it's made to add an actor quickly without opening the dialog.

**Comment by GitGianluc** [comment]
This might be quite confusing, but if you think it's OK… you are the boss! 😄

**Comment by URUWorks** [comment]
I convert it to discussion, and we are seeing opinions :)

**Reply by GitGianluc** [reply]
I'm sure @chenlung could be the deciding factor
